src/main/java/org/mskcc/cbio/oncokb/web/rest/CronJobController.java

```diff
@@ -274,6 +242,18 @@ public void analyzeUserUsage() throws IOException {
         log.info("User usage analysis completed!");
     }
 
+    /**
+     * {@code GET /wrap-token-stats}: Wrap token stats
+     *
+     * @throws IOException
+     */
+    @DeleteMapping(path = "/wrap-token-stats")
+    public void wrapTokenStats() throws IOException {
+        updateTokenStats();
+        s3Service.saveObject("oncokb", TOKEN_STATS_STORAGE_FILE_PREFIX + Instant.now().toString() + ".txt", createWrappedFile());
```

Comment:
What's the file name? Could you give me an example?

Reply:
`token-stats-2022-01-19T16:49:45.951685Z.txt` Here is a test file I ran earlier.

Reply:
We can design to fetch all tokenstates generated on previous day. Don't have to use instant.now(). So the file name would be easier to the eye, also more organized. Thought?

Reply:
Yeah, that would be better. Would just removing the time portion of the name and putting it one day before be okay? `token-stats-2022-01-19.txt` or just `2022-01-19.txt`.

Reply:
Let's keep the prefix